### PR TITLE
Remove Azure US East Config and Credentials

### LINKED ISSUE #197: Migrate to Azure Open AI  and Open AI dedicated account for Ask-Astro
Currently, we use a CSE dev account for Azure Open AI. And Open AI API key is not dedicated for Ask-Astro.

- [x] Create IT ticket for dedicated Azure Open AI and get permission from Microsoft Open AI
- [ ] Get access to separate prod and dev Open AI key for Ask-Astro
- [ ] Deploy the dev and prod with the above keys

### PR DESCRIPTION
**Pull request by davidgxue**
### Description
- We have now migrated Ask Astro credentials into a single account on Azure. All OpenAI model deployments now use the same Azure endpoint (US East 2) and credentials. Therefore, the US East credentials environment variable and the config field is no longer relevant.

### Tests
- New credentials already tested in dev servers. `AZURE_OPENAI_USEAST_PARAMS` and `AZURE_OPENAI_USEAST2_PARAMS` was set to the be same
- In this branch, I removed the environment variable for `AZURE_OPENAI_USEAST_PARAMS` and the service still worked as expected.

Related to Issue #197 (Azure OpenAI migration, not OpenAI)

### CODE REVIEW

**Review by Lee-W — APPROVED** [review]

File: api/ask_astro/clients/weaviate_.py [hunk 1]
```diff
@@ -3,19 +3,10 @@
...
 import weaviate
-from langchain.embeddings import OpenAIEmbeddings
 from langchain.vectorstores import Weaviate
 from weaviate import Client as WeaviateClient
 
-from ask_astro.config import AzureOpenAIParams, WeaviateConfig
-from ask_astro.settings import WEAVIATE_OPENAI_EMBEDDINGS_DEPLOYMENT_NAME, WEAVIATE_OPENAI_EMBEDDINGS_MODEL
-
-# Initialize OpenAI embeddings using the specified parameters.
-embeddings = OpenAIEmbeddings(
```
Comment (sunank200) [inline api/ask_astro/clients/weaviate_.py:14]: why is this removed?

Comment (davidgxue) [reply api/ask_astro/clients/weaviate_.py:14]: This is dead code. Look around in the repo. `embeddings` is never used anywhere. This make sense because the embedding model should only ever be used during ingestion. And that is setup in the `schema.json` file. Not here
